Asilo5/Ballroom-Beats-UI
# Aw landscape view

@spaceplesiosaur @Asilo5 
#### What's this PR do?
- Adds react-native-extended-stylesheet
- switches orientation in app.json from "portrait" to "default"
- adds EStyleSheet.build to app
- add @media (orientation: landscape) to components for styling
#### Where should the reviewer start?
- look at code additions in stylesheets
#### How should this be manually tested?
- open ios simulator, change orientation to landscape, refresh the screen and see changes
#### Any background context you want to provide?
- additional styling will need to be done for the dance to look good
